### ISSUE TITLE
Install KSCrash up app in Swift but not in Objective-C

### ISSUE THREAD

**Issue by joshdholtz** [kstenerud/KSCrash#110]
### Behavior
For some reason when installing `KSCrash` in a Swift app (using CocoaPods with frameworks), the app seems to lock up and requires this user to force close the app. The crash is still reported.

When using the same setup in an Objective-C project, the app does force close properly.

Both of the above were tested when the device **was not** connect to the debugger as if the app was running wild and free from the App Store.

**Sample Swift and Objective-C Project** - https://github.com/joshdholtz/KSCrash-Locking-Up-In-Swift-Example

### Objective-C
```objc
@implementation ViewController

- (void)viewDidLoad {
	[super viewDidLoad];

	KSCrash *crash = [KSCrash sharedInstance];
	[crash install];
}

- (IBAction)onClickBreakSomething:(id)sender {
	// The app crash properly here
	// Nothing unusual to see
	//
	// KSCrash will do its thing and capture this report
	// before the app force closes
	NSMutableArray *array = @[].mutableCopy;
	[array addObject:nil];
}


@end
```

### Swift
```swift
class ViewController: UIViewController {

	override func viewDidLoad() {
		super.viewDidLoad()

		let crash = KSCrash.sharedInstance()
		crash.install()
	}

	@IBAction func onClickBreakSomething(sender: AnyObject) {
		// The app will lock up here completely (even when not
		// running in debug). The button text stays highlighted
		// and will stay like this until the user force closes
		// the app
		//
		// KSCrash still capture this report but the app needs
		// to be force closed by the user
		let string: String! = nil
		string.lowercaseString
	}

}
```

**Comment by kstenerud** [comment]
Hi folks, I've been busy moving to a new country over the past month. Once I get a new computer tomorrow, I'll start updating KSCrash to work with the latest Xcode.

**Comment by joshdholtz** [comment]
Sounds great! I was wanting to look into this issue myself to help you out but I have zero idea where to go with it :innocent:

**Comment by kstenerud** [comment]
OK, finally got a macbook and tested this, but I can't get the project you linked to lock up, either on the simulator or on device. I press "Break something", the app quits, and I've verified that it generates a report.

**Comment by joshdholtz** [comment]
@kstenerud That is super weird! I did actually run it on my iPhone 6 without the debugger attached (physically disconnected from my computer) and it locked up there too. I will try again.

And just to verify, you ran the Swift project? The Objective-C one worked fine. The Swift one was the one that locked up on  me.

**Comment by kstenerud** [comment]
Yes, I loaded the swift workspace, loaded it onto my ipod 5th gen, and ran it. No lockup. I wonder if maybe it has something to do with 64 bit arm? Does it work when you use the simulator?

**Comment by joshdholtz** [comment]
@kstenerud It does appear to crash in the simulator for me but not on my iPhone 6.

**Comment by kstenerud** [comment]
Damn... That means I need to source a 64-bit device again so I can test this.

**Comment by joshdholtz** [comment]
@kstenerud I will verify one more time this afternoon for you before you start trying to find one :innocent:

**Comment by joshdholtz** [comment]
@kstenerud Definitely still locks up my phone. Had to force shut down my phone because it looked almost the whole thing up :innocent:

**Comment by joshdholtz** [comment]
Let me know if there is anything I can do to help with this! If you can point me in a direction of where to look / who to debug, I can give it a try :blush:

**Comment by kstenerud** [comment]
Actually I've gotten it to hang on the simulator. It's hanging up while trying to introspect an objective-c object. You can tell when this happens because the json report file will end abruptly like so:

                },
                "notable_addresses": {
                    "rax": {
                        "address": -1,
                        "type": "objc_object"

For now it's best to disable introspection until I can figure out what's breaking.

**Comment by naftaly** [comment]
Swift installation has been supported and documented since 2.0. See the README for Swift integration examples.
